src/Compilers/CSharp/Portable/Binder/ContextualAttributeBinder.cs

```diff
@@ -64,5 +66,10 @@ internal Symbol AttributeTarget
                 return _attributeTarget;
             }
         }
+
+        protected override CSDiagnosticInfo SingleTypeNameNotFound(string whereText, DiagnosticBag diagnostics, Location location, AssemblySymbol forwardedToAssembly)
```

Comment:
I did notice that with one of the emit tests. Can you perhaps show a test exploiting this as all other tests seem OK with the change?